statler_site/templates/play-detail-page.html

```diff
@@ -64,6 +66,11 @@
       <div id="details"></div>
       <div style="height: 200px; border-style: solid;">
         Review post box will go here.
+        <form action="{% url 'api:play-reviews' play_id %}" method="post">
+          {% csrf_token %}
```

Comment:
Using form encoding instead of JSON here feels really dirty. The whole idea of separating the API and the site is to make the API flexible and independent of messy HTML implementation stuff. I'm pretty sure we should be able to create JSON on the web side with less than ten lines of JavaScript, and I think the API consistency is worth it.